.circleci/scripts/release-commit-verison-bump.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Typo in file name: `verison` -> `version`.

It appears to be correctly spelled in the CircleCI config, but we should review all references.

Reply:
Oops :facepalm: Good catch! Fixed.